docs/docs-content/release-notes.md

```diff
@@ -10,7 +10,27 @@ sidebar_custom_props:
   icon: "audits"
 tags: ["release-notes"]
 ---
- 
+
+## October 3, 2023 - Release 4.0.15
+
+This release contains enhancements and minor bug fixes.
+
+## Enhancements
+
+- Configurable options to specify charging for Azure Kubernetes Service (AKS) cluster control planes are now available in the Kubernetes `values.yaml` pack. These options are based on Service Level Agreements (SLA) that Azure provides. The ability to set these options in the Kubernetes YAML file allows you to embed options and overrides in cluster profiles.
+
+  Use the `managedControlPlane.sku` parameter to specify Azure recommended settings: `Standard` for production clusters and `Free` for non-production or small clusters.
```

Comment:
@ritawatson it feels like we are putting too much information here. Can we instead add a small heading in the Azure architecture page, document the feature, and link to it here? The Jira ticket has the YAML values, which would be good to capture.